command/pac/server/server.go

```diff
@@ -38,6 +38,10 @@ func (c *command) runE(cmd *cobra.Command, _ []string) (cmdErr error) {
 	}
 	logger := stdlog.New(c.logConfig)
 
+	defer func() {
+		logger.Close()
+	}()
```

Comment:
Just `defer logger.Close()`?

Reply:
Actually maybe we should be printing the error to stderr

Reply:
👍 👍